packages/orbit-components/src/Stack/types.d.ts

```diff
@@ -52,4 +52,5 @@ export interface Props extends CommonProps, Common.Globals {
   readonly as?: string;
```

Comment:
When adding `ariaLabel` bcs of `as`, maybe I would also change type of `as` to `readonly as?: Common.Component;`. wdyt?